scripts/all-tests.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 # Violations thresholds for failing the build
 export LOWER_PYLINT_THRESHOLD=1000
-export UPPER_PYLINT_THRESHOLD=5335
+export UPPER_PYLINT_THRESHOLD=6200
```

Comment:
BTW: on master with this merged, pylint is at 5827, so we overestimated the headroom we'd need for this.